content/containers.md

```diff
@@ -51,4 +51,4 @@ Containers may be restarted for several reasons, such as:
 
 If constant uptime is a core requirement, we recommend running your app on more than 1 container. Since periodic system updates and container restarts can be expected on Galaxy, this can help prevent unwanted downtime. When an app runs on multiple containers, traffic can be routed elsewhere when a single container becomes unresponsive.
 
-Please note that, if you do choose to run a 1 container app, downtime will be unavoidable in the event of a container restart. Due to the need for system maintenance, we cannot guarantee that containers will not be restarted.
\ No newline at end of file
+Please note that, if you do choose to run a 1 container app, downtime will be unavoidable in the event of a container restart or a new deployment. Due to the need for system maintenance, we cannot guarantee that containers will not be restarted.
```

Comment:
Sorry - I'm not clear on the meaning here. If you choose to run a 1 container app (contrary to our suggestions) and have a new deployment, there will be downtime. If you're thinking that the line or meaning should be changed in a way that I'm missing, though, please clarify (I think I'm missing something myself).

Reply:
When you deploy a new version (or we replace machines, or you scale your app, etc), we do our best to start new containers before stopping old containers.  These routine operations should not lead to downtime.  What will lead to downtime is an unpredictable failure: the container crashing, a machine suddenly failing (which rarely happens), an availability zone losing network connectivity, etc.  But ordinary events that are under our control should not cause outages.

That said: yes, people who want to avoid downtime should absolutely run multiple containers.  But that's not the same as saying ordinary app deploys cause downtime.

(I'm suggested that we consider a different policy for single-container apps in order to allow people to rely on at-most-one behavior for single-container apps, as well as to help highlight to users that relying on a single-container app for 100% uptime is a bad idea. But that is not our current policy.)

Reply:
I understand now, and I can help to highlight that suggestion; I'll make a PR about this.